### PR TITLE
test: update white list and correct links in docs closes #832 #833

### LINKED ISSUE #832: An in-range update of textlint-rule-no-dead-link is breaking the build 🚨
## Version **3.4.0** of [textlint-rule-no-dead-link](https://github.com/textlint-rule/textlint-rule-no-dead-link) was just published.

<table>
  <tr>
    <th align=left>
      Branch
    </th>
    <td>
      <a href="/hoodiehq/hoodie/compare/greenkeeper%2Ftextlint-rule-no-dead-link-3.4.0">Build failing 🚨</a>
    </td>
  </tr>
  <tr>
    <th align=left>
      Dependency
    </td>
    <td>
      textlint-rule-no-dead-link
    </td>
  </tr>
  <tr>
    <th align=left>
      Current Version
    </td>
    <td>
      3.3.1
    </td>
  </tr>
  <tr>
    <th align=left>
      Type
    </td>
    <td>
      devDependency
    </td>
  </tr>
</table>

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.

textlint-rule-no-dead-link is a devDependency of this project. It **might not break your production code or affect downstream projects**, but probably breaks your build or test tools, which may **prevent deploying or publishing**.

<details>
<summary>Status Details</summary>

- ❌ **continuous-integration/travis-ci/push** The Travis CI build could not complete due to an error [Details](https://travis-ci.org/hoodiehq/hoodie/builds/317972587?utm_source=github_status&utm_medium=notification)
</details>

---

<details>
<summary>FAQ and help</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html). If those don’t help, you can always [ask the humans behind Greenkeeper](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>

---

Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

### PR DESCRIPTION
**Pull request by fozy81**

### CODE REVIEW

**Comment by fozy81** [comment]
closes #833 #832

**Comment by fozy81** [comment]
updated the PR to remove dependency on awesome_bot and set dead link tests to warnings only so won't break the build.

File: .travis.yml [hunk 1]
```diff
@@ -3,22 +3,17 @@ language: node_js
 notifications:
   email: false
 node_js:
-  - 4
+  - 6
```
Comment (gr2m) [inline .travis.yml]: Does it fail in Node 4? Not a big deal, I planned to suggest to align Node dependency with hapi anyway (which is 8 now), and I think requiring 6 is reasonable, just wondering

**Comment by fozy81** [comment]
The upping the node version was suggested as a [fix](https://github.com/textlint-rule/textlint-rule-no-dead-link/issues/81#issuecomment-353582093) to a problem with using textlint-rule-no-dead-link but now that I've removed this dependency (also removed it from package.json just now). And I've returned node version 4. So leave it with you to update to version 8. Sorry - went through a few options trying to get things to work this morning and forgot I'd changed the node version.

**Review by gr2m — APPROVED** [review]
thank you :)

**Pull request merged**